<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagina_ciudad</class>
 <widget class="QWidget" name="pagina_ciudad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="boton_editar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #FF6A36;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>editar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_titulo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Añadir nueva ciudad</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>171</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Layout_ciudad">
    <item>
     <widget class="QLabel" name="label_estatico_ciudad">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ciudad</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ciudad">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 13px;
    color: #606266;
    min-height: 18px;
}

QLineEdit:hover {
    border-color: #c0c4cc;
}

QLineEdit:focus {
    border-color: #409eff;
}

QLineEdit:disabled {
    background-color: #f5f7fa;
    color: #c0c4cc;
    border-color: #e4e7ed;
}

QLineEdit:read-only {
    background-color: #f5f7fa;
    color: #909399;
}</string>
      </property>
      <property name="inputMask">
       <string>aaaaaaaaaaaaaaaaaaaa</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="boton_agregar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>Añadir ciudad</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>171</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Layout_codigo">
    <item>
     <widget class="QLabel" name="label_estatico_codigo">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Codigo </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_codigo">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 13px;
    color: #606266;
    min-height: 18px;
}

QLineEdit:hover {
    border-color: #c0c4cc;
}

QLineEdit:focus {
    border-color: #409eff;
}

QLineEdit:disabled {
    background-color: #f5f7fa;
    color: #c0c4cc;
    border-color: #e4e7ed;
}

QLineEdit:read-only {
    background-color: #f5f7fa;
    color: #909399;
}</string>
      </property>
      <property name="inputMask">
       <string>&gt;AAA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="boton_cancelar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #FF6A36;
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_ciudad">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    border-radius: 15px;
    gridline-color: #e0e0e0;
    outline: none;
    font-size: 13px;
    color: #606266;
}

QTableWidget::item {
    padding: 8px;
    border-bottom: 1px solid #f0f0f0;
}

QTableWidget::item:selected {
    background-color: #409eff;
    color: white;
}

QTableWidget::item:hover {
    background-color: #f5f7fa;
}

/* Header con bordes redondeados solo arriba */
QHeaderView::section:first {
    border-top-left-radius: 15px;
}

QHeaderView::section:last {
    border-top-right-radius: 15px;
}

QHeaderView::section {
    background-color: #1061C4;
    color: white;
    font-weight: bold;
    font-size: 13px;
    padding: 8px 12px;
    border: none;
    border-right: 1px solid #0a4a9d;
}

QHeaderView::section:hover {
    background-color: #0a4a9d;
}

QHeaderView::section:pressed {
    background-color: #083a7a;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>179</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Codigo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
